atom/browser/net/atom_cert_verifier.h

```diff
@@ -34,7 +34,7 @@ class AtomCertVerifier : public net::CertVerifier {
   void SetVerifyProc(const VerifyProc& proc);
 
   const VerifyProc verify_proc() const { return verify_proc_; }
-  AtomCTDelegate* ct_delegate() const { return ct_delegate_; }
+  AtomCTDelegate* ct_delegate() const { return ct_delegate_.get(); }
```

Comment:
As a side issue, returning a raw pointer to a managed resource like this worries me a little... but in this case we have a cheap workaround, the only customer of this method is `CertVerifierRequest` which is already a `friend`, so we could move `ct_delegate()` a few lines down here into the `protected` section of the class

Reply:
:+1: